concourse/README.md

```diff
@@ -0,0 +1,103 @@
+# Concourse
+The concourse plan _only_ provides user's access to the standalone [concourse binary](https://concourse.ci/binaries.html). It is provided with the intention that any user's who need to run concourse will write their own plan that depends on this package and provide their own runtime management.
+
+Concourse is a complex piece of software with multiple ways in which it can be configured to run. As such we provide this package to enable you to wrap and automate your preferred deployment style.
+
+## Examples
+Below we have provided an example configuration for deploying Concourse with a wrapper plan.
```

Comment:
Rather than 'wrapper plan', I thought we had come up with different terminology to describe these kinds of plans at the Seattle Summit. I don't recall what the terminology was, I'll update here if I can remember.

Reply:
Oh weird I guess my comment got lost somehow. Basically, yes you are correct but also we had a conversation after the summit in which we decided we didn't need a name for this pattern because its just THE way you use a plan that only provides a binary to disk. I've updated the pr to take out the "wrapper plan" bit from this sentence.